styles.css

```diff
@@ -88,15 +88,19 @@ header {
 }
 
 #social-icons {
+  color: #172b4d;
   list-style-type: none;
   padding: 0;
   margin: 0;
   display: flex;
   justify-content: flex-start;
-  color: #344563;
 }
 
 .icon {
-  color: #505f79;
+  color: grey;
   margin-right: 12px;
 }
+
+#icon-color {
+  color: grey;
+}
```

Comment:
[OPTIONAL]

- In other to maintain the design, let's try to use the same image as used in Figma to make our design close to the original design.